src/hooks/useV2Trade.ts

```diff
@@ -70,26 +70,28 @@ export function useV2TradeExactIn(
 
 /**
  * Returns the best trade for the token in to the exact amount of token out
+ * @param forcedMaxHops Sets max hops to `forcedMaxHops` regardless of useUSerSingleHopOnly
```

Comment:
strange to pull from state and have an override, when you can just take the max hops number as an argument and pass in 2 in one location and the result of state in another